bigquery/google/cloud/bigquery/table.py

```diff
@@ -619,54 +619,62 @@ def _set_properties(self, api_response):
             cleaned['expirationTime'] = float(cleaned['expirationTime'])
         self._properties.update(cleaned)
 
-    def _build_resource(self, fields=[]):
+    def _populate_expires_resource(self, resource):
+        value = _millis_from_datetime(self.expires)
+        resource['expirationTime'] = value
+
+    def _populate_partitioning_type_resource(self, resource):
+        resource['timePartitioning'] = self._properties['timePartitioning']
+
+    def _populate_view_use_legacy_sql_resource(self, resource):
+        if 'view' not in resource:
+            resource['view'] = {}
+        resource['view']['useLegacySql'] = self.view_use_legacy_sql
+
+    def _populate_view_query_resource(self, resource):
+        if 'view' not in resource:
+            resource['view'] = {}
+        resource['view']['query'] = self.view_query
+
+    def _populate_schema_resource(self, resource):
+        if len(self._schema) == 0:
```

Comment:
Will `_schema` never be `None`?

Reply:
At this time, the schema will never be None. If it is set to None, the setter function sets it to an empty tuple. A later pull request will add the ability for the schema to be set to None.